bundle.js

```diff
@@ -0,0 +1,641 @@
+(function () {
```

Comment:
Did you mean to commit this? (and build/protections.js)

Reply:
Not really but there's a lot of mess like this still needing cleaning :)